types/mdast/tslint.json

```diff
@@ -1,6 +1,3 @@
 {
-    "extends": "dtslint/dt.json",
-    "rules": {
-        "no-redundant-undefined": false
```

Comment:
removes global disable, as recommended in https://github.com/DefinitelyTyped/DefinitelyTyped#linter-tslintjson
Updating this, clears updates to the types from being flagged as `Check Config` https://github.com/DefinitelyTyped/DefinitelyTyped/labels/Check%20Config

Reply:
Hey, that change is OK. The others in index.d.ts are not IMO, they are perfectly legit and should not rise any warnings under modern. Is that OK for you folks to leave them or do you want to handle 4.4 strictness differently?
https://github.com/microsoft/TypeScript/issues/44403

Reply:
@peterblazejewicz are you saying that if `exactOptionalPropertyTypes` is added to the tsconfig, these ignores will not be needed?

Reply:
no, those rules are for `TSLint`, IMO have no direct relation to compiler. What I meant, those were added by maintainer for compatibility with 4.4 (if someone use that options referencing this DT package). Owners of package decide, if that option should be supported.
Here I meant: one does not need those tslint rules here. Are you getting errors here?

Reply:
> Owners of package decide, if that option should be supported

The intent is that `exactOptionalPropertyTypes` should be supported.

> Here I meant: one does not need those tslint rules here. Are you getting errors here?

Yes, tslint throws an error if the ignores are not included, which is why I added them.

Reply:
I cannot see this error, this is why I asked, what's your intent here. It's enough to trim down tslint config (checkout PR, revert index. ts and run all tests). What's your setup?
sorry to bother you folks, thx!

Reply:
> What's your setup?

I did `npm install && npm test mdast` with node v16.10.0 and npm 7.24.0 on a linux machine.
Which I assume is fairly similar to your setup/the CI setup?

The only other thing I can think of, is since there isn't a lockfile npm may hold on to older version of dependencies?
Should I also be doing `rm -rf node_modules` as part of this process as well?

> I cannot see this error

:thinking: strange

Reply:
let me check on different environment (Debian), one sec

Reply:
@ChristianMurphy do you have an editor plugin maybe?

Reply:
I'll resolve this conversation, merge at your discretion, thx!

Reply:
After `rm -rf node_modules` and installing all dependencies again, the error went away.
I think I was on an old version of dtslint locally.

Thanks for the prompt to look in more @peterblazejewicz!